.config/mypy/mypy.ini

```diff
@@ -0,0 +1,10 @@
+[mypy]
+
+[mypy-IPython]
+ignore_missing_imports = True
+
+[mypy-colorama]
+ignore_missing_imports = True
+
+[mypy-traitlets.config.loader]
+ignore_missing_imports = True
```

Comment:
Could you add the following lines? 
```
[mypy-scapy.modules.six,scapy.modules.winpcapy]
ignore_errors = True
```

They disable errors reported in modules that we will likely not bother to type.

Reply:
Done